packages/app/src/components/DimensionsPanel/Dialogs/AddToLayoutButton/AddToLayoutButton.js

```diff
@@ -1,20 +1,16 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
-import Button from '@material-ui/core/Button'
-import MenuItem from '@material-ui/core/Button'
-import { withStyles } from '@material-ui/core/styles'
 import {
     getAvailableAxes,
     getAxisNameByLayoutType,
     getLayoutTypeByVisType,
 } from '@dhis2/analytics'
+import { SplitButton, Menu, MenuItem, Button } from '@dhis2/ui-core'
 
-import Menu from './Menu'
 import { sGetUiActiveModalDialog, sGetUiType } from '../../../../reducers/ui'
 import { acAddUiLayoutDimensions } from '../../../../actions/ui'
 
-import styles from './styles/AddToLayoutButton.style'
 import i18n from '@dhis2/d2-i18n'
 
 export class AddToLayoutButton extends Component {
```

Comment:
We can convert this into a functional component.
Apparently we don't need to use state as `onClose` and `onToggle` are not used anymore?

Reply:
Yes, you're right, `onClose` and `onToggle` aren't used anymore. And all components touched by this PR should be converted to functional with css modules 👍

Reply:
Fixed 🚀